src/embed_tests/TestOperator.cs

```diff
@@ -23,6 +24,120 @@ public void Dispose()
+        public struct OwnInt
+        {
+            private int _value;
+
+            public int Num => _value;
+
+            public OwnInt()
+            {
+                _value = 0;
+            }
+
+            public OwnInt(int value)
+            {
+                _value = value;
+            }
+
+            public static OwnInt operator -(OwnInt p1, OwnInt p2)
+            {
+                return new OwnInt(p1._value - p2._value);
+            }
+
+            public static OwnInt operator +(OwnInt p1, OwnInt p2)
+            {
+                return new OwnInt(p1._value + p2._value);
+            }
+
+            public static OwnInt operator *(OwnInt p1, OwnInt p2)
+            {
+                return new OwnInt(p1._value * p2._value);
+            }
+
+            public static OwnInt operator /(OwnInt p1, OwnInt p2)
+            {
+                return new OwnInt(p1._value / p2._value);
+            }
+
+            public static OwnInt operator %(OwnInt p1, OwnInt p2)
+            {
+                return new OwnInt(p1._value % p2._value);
+            }
+
+            public static OwnInt operator ^(OwnInt p1, OwnInt p2)
+            {
+                return new OwnInt(p1._value ^ p2._value);
+            }
+
+            public static bool operator <(OwnInt p1, OwnInt p2)
+            {
+                return p1._value < p2._value;
+            }
+
+            public static bool operator >(OwnInt p1, OwnInt p2)
+            {
+                return p1._value > p2._value;
+            }
+
+            public static bool operator ==(OwnInt p1, OwnInt p2)
+            {
+                return p1._value == p2._value;
+            }
+
+            public static bool operator !=(OwnInt p1, OwnInt p2)
+            {
+                return p1._value != p2._value;
+            }
+
+            public static OwnInt operator |(OwnInt p1, OwnInt p2)
+            {
+                return new OwnInt(p1._value | p2._value);
+            }
+
+            public static OwnInt operator &(OwnInt p1, OwnInt p2)
+            {
+                return new OwnInt(p1._value & p2._value);
+            }
+
+            public static bool operator <=(OwnInt p1, OwnInt p2)
+            {
+                return p1._value <= p2._value;
+            }
+
+            public static bool operator >=(OwnInt p1, OwnInt p2)
+            {
+                return p1._value >= p2._value;
+            }
+        }
+
+        // Codec for mock class above.
+        public class OwnIntCodec : IPyObjectDecoder
+        {
+            public static void Setup()
+            {
+                PyObjectConversions.RegisterDecoder(new OwnIntCodec());
+            }
+
+            public bool CanDecode(PyType objectType, Type targetType)
+            {
+                return objectType.Name == "int" && targetType == typeof(OwnInt);
+            }
+
+            public bool TryDecode<T>(PyObject pyObj, out T? value)
```

Comment:
Please add `#nullable enable` to the file and fix the warnings or remove this nullable annotation.